docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,146 @@
+        </section>
+    </div>
+{% endblock %}
+```
+At the line 8, you see the import Twig function, that is calling the `user/registration_content_form.html.twig` file. So create this file along the previous file, and fill it with these Twig tags:
+```html
+{% macro display_form(form) %}
+    {{ form_start(form) }}
+
+    {% for fieldForm in form.fieldsData %}
+        {% set fieldIdentifier = fieldForm.vars.data.fieldDefinition.identifier %}
+
+        {% if fieldIdentifier == 'first_name' or fieldIdentifier == 'last_name' %}
+            {% if fieldIdentifier == 'first_name' %}
+                <div class="row">
+            {% endif %}
+            <div class="col-md-6">
+                <div class="field-name">
+                    <label class="required">{{ fieldForm.children.value.vars.label }}:</label>
+                </div>
+                {{ form_errors(fieldForm.value) }}
+                {{ form_widget(fieldForm.value, {
+                    'contentData': form.vars.data
+                }) }}
+            </div>
+            {% if fieldIdentifier == 'last_name' %}
+                </div>
+            {% endif %}
+        {% endif %}
+
+        {% if fieldIdentifier == 'user_account' %}
+            <div class="row">
+                <div class="col-md-6">
+                    {{ form_widget(fieldForm.value, {
+                        'contentData': form.vars.data
+                    }) }}
+                </div>
+            </div>
+        {% endif %}
+
+        {%- do fieldForm.setRendered() -%}
+    {% endfor %}
+
+    <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+            {{ form_widget(form.publish, {'attr': {
+                'class': 'btn btn-block btn-primary'
+            }}) }}
+        </div>
+    </div>
+
+    {{ form_end(form) }}
+{% endmacro %}
+```
+If you want to check the look of your new page, first clear the cache, then get your <yourdomain>/register URL.
+
+### Create the confirmation page template
+Now create a file `app/Resources/views/user/registration_confirmation.html.twig` along the previous files.
+
+Put these Twig tags in the file:
+```html
+{% extends pagelayout %}
+
+{% block page_head %}
+  {% set title = 'Registerstration complete'|trans %}
+
+  {{ parent() }}
+{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <section class="user-form-confirmation col-md-4 col-md-offset-4">
+      <h2>{{ 'Registration completed'|trans }}</h2>
+
+      <div class="row confirmation-label">
+        {{ 'You are all set up and ready to go'|trans }}
+      </div>
+
+      <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+          <button type="button" class="btn btn-block btn-primary"
+                  onclick="window.location='{{ path('login') }}';">{{ 'Log in'|trans }}</button>
+        </div>
+      </div>
+    </section>
+  </div>
+{% endblock %}
+
+```
+
+Now you are able to customize the user registration of your website.
```

Comment:
Not clear to me - I think we just did customize it, didn't we?

Also, I think here's the right place for the screenshot.